CMakeLists.txt

```diff
@@ -51,6 +51,8 @@ endif()
 
 ez_pull_platform_vars()
 
+include("CMakeUtils/ezUtilsPlatformVars")

```

Comment:
This line should not be necessary. All files in this folder are automatically included.

Reply:
The entire point of this include is to make sure that the EZ_CMAKE_PLATFORM ... vars are already defined.
In fact I don#t want this file to be auto included. Where do I have to place it, so that it is not auto included?

Reply:
nvm found it.

Reply:
Literally into any other folder, even sub folders would work.